lib/lib-javadoc/src/main/resources/controllersHtmlTemplate.html

```diff
@@ -7,7 +7,7 @@
 		<div id="main">
 			<div id="mainWrapper" class="ym-wrapper">
 				<div class="ym-wbox">
-					<div id="rootPanel" class="ym-grid linearize-level-1">
+					<div id="rootPanel" class="ym-grid linearize-level-1 allServicesList">

```

Comment:
styling that was applied to #rootPanel (which incorrectly applied to elements on individual service pages) is now applied to div.allServicesList, which only occurs here